### ISSUE TITLE
Ferocious Bite scaling seems wrong

### ISSUE THREAD

**Issue by Jerrythemagicdog** [SunstriderEmu/BugTracker#1343]
**Describe the bug**
Ferocious Bite seems to be doing too little damage, the scaling with attack power could be wrong or the energy spent bonus not affecting it. 
Need sources and research here to confirm.

**Comment by blblblaz** [comment]
according to this formula from wow wiki -
Ferocious Bite Rank 6: Damage = ( AP * 0,1526 + [energy - 35] * 4,1 + 935 to 968 )  - 
the average damage of a non-crit 5 combo point untalented Ferocious Bite with full energy should be around 1735~ on a target with 0 armor

https://imgur.com/a/eaccHHp -this is a Ferocious Bite with 100 energy 

https://imgur.com/FSqmC6R - this is a Ferocious Bite with 40 energy

it seems the energy scaling is too high and the overall damage is probably too low

fuck feral druid

**Comment by ramadansteve** [comment]
> according to this formula from wow wiki -
> Ferocious Bite Rank 6: Damage = ( AP * 0,1526 + [energy - 35] * 4,1 + 935 to 968 ) -
> the average damage of a non-crit 5 combo point untalented Ferocious Bite with full energy should be around 1735~ on a target with 0 armor
> 
> https://imgur.com/a/eaccHHp -this is a Ferocious Bite with 100 energy
> 
> https://imgur.com/FSqmC6R - this is a Ferocious Bite with 40 energy
> 
> it seems the energy scaling is too high and the overall damage is probably too low

https://youtu.be/GYUbaSCjCTU?t=401
mr. fat cow here hits a destruction warlock with _**4**_ CP 41 energy FB for 1318.
https://youtu.be/DMaCg_US5pU?t=1893
and here mr. beancounter hits an arcane mage with mage armor (and spellstolen MOTW +340 armor) hits a 5 CP 40 energy FB for 826 (quiet fart sound)
notice how agility idol buff was present in both cases.
also, consider that mr. beancounter's gear is most likely better than mr. chrille's thus more AP. plus he was wearing berserker's call trinket and as opposed to crit BM.
now, lets assume mr. fat cow's gear was the same exact gear as that of mr. beancounter's except the trinket, that would put him at 3341 AP with agi buff up.
lets use this value in the formulae given: (3341 * 0,1526 + 6*4,1 + 766-799) let's also assume mr. fat cow was extremely lucky and hit the highest roll (799). according to this formulae his bite should hit for 1333 on a target with 0 armor, which the warlock isn't and having this much arp is impossible for the feral. he could have up to 560 arp but that would mean he had less AP.

tldr: 
first of all this formula seems off
secondly, the difference between 4cp 41 energy 1318 and 5cp 40energy 829 seems huge to me. i would imagine the mage had a bit more armor bcs of motw and i had a bit less ARP than chrille in the vid but also more AP. this tells me that currently on endless FB hits for about 40% weaker than it should

**Comment by Jerrythemagicdog** [comment]
Looks to be correct now with the new formula.
